packages/itwinui-css/src/style/global.scss

```diff
@@ -35,31 +35,38 @@ html {
   }
 }
 
-.iui-body {
-  background-color: var(--iui-color-background-2);
+:where(.iui-root) {
   color: var(--iui-text-color);
   font-size: $iui-font-size;
   line-height: $iui-line-height;
 
-  @include iui-scrollbar;
   @include iui-font-family;
 
-  &.iui-no-webfonts {
-    font-family: $iui-sans-fallback;
+  // reset margin and apply background only for <body> element
+  @at-root body#{&} {
+    margin: 0;
+    background-color: var(--iui-color-background-2);
   }
-}
-
-[class*='iui-'],
-[class*='iui-'] * {
-  // Text highlight
-  @include iui-text-selection;
 
-  // Scrollbar
-  @include iui-scrollbar;
+  // border-box on all iui- elements and their descendants under iui-root, excluding any nested iui-root elements
+  :where([class*='iui-']) {
+    &:where(:not(.iui-root)),
+    * {
+      &,
```

Comment:
no it's `*, *::before, *::after`

https://sass-lang.com/documentation/style-rules/parent-selector

Reply:
I'm sorry, I'm still a little confused. Are both these code blocks equivalent?
#### Current
```scss
:where([class*='iui-']) {
  &:where(:not(.iui-root)),
  * {
    &,
    &::before,
    &::after {
      box-sizing: border-box;
    }
  }
}
```

#### New
```scss
:where([class*='iui-']) {
  &:where(:not(.iui-root)),
  *,
  *::before,
  *::after {
    box-sizing: border-box;
  }
}
```

Reply:
Close but `&` also includes `where(:not(.iui-root))` so it will also add `::before` and `::after` to that.

```scss
:where([class*='iui-']) {
  &:where(:not(.iui-root)),
  &:where(:not(.iui-root))::before,
  &:where(:not(.iui-root))::after
  *,
  *::before,
  *::after {
    box-sizing: border-box;
  }
}
```

I would suggest you again to run `yarn build` and look at the compiled css.